tests/addToCart.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Login', () => {
+  it('fill in valid data and login', async () => {
+    await browser.url(`https://www.saucedemo.com/`);
```

Comment:
It can be 

```
await browser.url(`/`);
```

because you already added baseUrl in config - https://github.com/slavian333/automation_tests/pull/2/files#diff-f297ad33212595210fbfc7c32d334f7025fe8eee14cc4179d2527c3f4543035fR99